FrontEnd/src/components/ProfilePage/FormComponents/AdditionalInfo.js

```diff
@@ -0,0 +1,114 @@
+import css from './FormComponents.module.css';
+import { useState, useEffect } from 'react';
+
+import HalfFormField from './FormFields/HalfFormField';
+import TextField from './FormFields/TextField';
+
+const LABELS = {
+    'foundationYear': 'Рік заснування',
+    'companySize': 'Розмір компанії',
+    'topClients': 'Топ клієнти',
+    'passedAudit': 'Пройдений аудит',
+};
+
+const AdditionalInfo = (props) => {
+    const [user, setUser] = useState(props.user);
+    const [foundationYearError, setFoundationYearError] = useState(null);
+    const maxLength = 1000;
+
+    useEffect(() => {
+        props.CurrentFormNameHandler('AdditionalInfo');
+    }, []); 
+
+    const onUpdateTextAreaField = e => {
+        if (e.target.value.length <= maxLength)
```

Comment:
Consider using [react-hook-form](https://www.npmjs.com/package/react-hook-form). It might simplify work with validation and forms at all

Reply:
I have never used this tool, so I implemented as I knew. Is this a problem? Or did you just recommend for future use?